### PR TITLE
tests and docs for include

### LINKED ISSUE #11: Document and test `include`

### PR DESCRIPTION
**Pull request by DanilloCorvalan**
fixes #11 
@shawninder

### CODE REVIEW

**Comment by shawninder** [comment]
Docs don't mention that you can provide an Object _thing_

File: README.md [hunk 1]
```diff
@@ -358,8 +358,24 @@ var hash = require('vigour-util')
...
 
-### include
-Docs coming soon
+<!-- VDOC.jsdoc include -->
+<!-- DON'T EDIT THIS SECTION (including comments), INSTEAD RE-RUN `vdoc` TO UPDATE -->
+#### var included = include(target, thing)
+
+Distinctly adds one or multiple items (in an object or array) to a target array. Doesn't duplicate items.
+- **target** (*Array*) - target array
+- **thing** - the thing to be included in the array. If object or array, it will go through it and add all values to the target array.
```
Comment (DanilloCorvalan) [inline README.md:367]: @shawninder

**Comment by shawninder** [comment]
@marcusbesjes Can you check that this covers everything?

**Comment by shawninder** [comment]
This is only docs and tests so I'm going to merge it and @marcusbesjes can check it whenever he has time

**Pull request merged**